CHANGELOG.md

```diff
@@ -1433,6 +1433,9 @@ Released 2018-09-13
 [`range_plus_one`]: https://rust-lang.github.io/rust-clippy/master/index.html#range_plus_one
 [`range_step_by_zero`]: https://rust-lang.github.io/rust-clippy/master/index.html#range_step_by_zero
 [`range_zip_with_len`]: https://rust-lang.github.io/rust-clippy/master/index.html#range_zip_with_len
+[`rc_borrows`]: https://rust-lang.github.io/rust-clippy/master/index.html#rc_borrows
```

Comment:
do you think it's worth merging these lints into a single "redundant_allocation" lint?

Reply:
sure, will do